README.md

```diff
@@ -165,3 +165,46 @@ Read more details in the dedicated [README](/Example/README.md).
 ### Dependencies
 
 * [node-qrcode](https://github.com/soldair/node-qrcode)
+
+### Integrating react-native-qrcode-svg with React Native versions below 0.75
+This library works seamlessly with React Native versions 0.75 and above. However, if you're using an older version of React Native (below 0.75), you might need to apply a custom transformation to your project's metro.config.js file to ensure compatibility with the TextEncoder API.
+
+Here's how to configure the transformer for both Expo and React Native projects:
+
+#### Setting Up the Transformer:
+Make sure your project has a `metro.config.js` file. If not, create one at the root of your project.
+
+```js
+const { getDefaultConfig } = require("expo/metro-config");
```

Comment:
here there's usage of `expo/metro-config` inside next code snippet there's `@react-native/metro-config` which can be confusing, let's unify it to `@react-native/metro-config` in both with a note "If you're using Expo use instead of `@react-native/metro-config` use `expo/metro-config`" - this part of the API surface AFAIK is the same 👍